### PR TITLE
[SPARK-44986][DOCS] There should be a gap at the bottom of the HTML

### PR DESCRIPTION
**Pull request by panbingkun**
### What changes were proposed in this pull request?
The pr aims to add gap at the bottom of the HTML.

### Why are the changes needed?
The old document style has good white space at the bottom, but the latest document has lost this piece, which looks unattractive and borderless.
image

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Manual testing.
```
SKIP_API=1 bundle exec jekyll serve --watch
```

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

**Comment by panbingkun** [comment]
Before:
image

After:
image

**Comment by zhengruifeng** [comment]
cc @allanf-db @HyukjinKwon @allisonwang-db

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
Please don't mix `2.align example order(Python -> Scala/Java -> R)` in this PR.

File: docs/quick-start.md [hunk 1]
```diff
@@ -470,19 +470,19 @@ Congratulations on running your first Spark application!
 * For an in-depth overview of the API, start with the [RDD programming guide](rdd-programming-guide.html) and the [SQL programming guide](sql-programming-guide.html), or see "Programming Guides" menu for other components.
 * For running applications on a cluster, head to the [deployment overview](cluster-overview.html).
 * Finally, Spark includes several samples in the `examples` directory
-([Scala]({{site.SPARK_GITHUB_URL}}/tree/master/examples/src/main/scala/org/apache/spark/examples),
+([Python]({{site.SPARK_GITHUB_URL}}/tree/master/examples/src/main/python),
```
Comment (allisonwang-db) [inline docs/quick-start.md:473]: Are there any other places where the Python one is not the first example?

**Review by allisonwang-db — COMMENTED** [review]
Thanks for the adjustment! Let's also update the PR title to include the example order updates.

**Comment by zhengruifeng** [comment]
> 

+1, it is irrelevant

**Comment by panbingkun** [comment]
Okay, Let me split it into a new separate PR, because I found similar order issues on other pages.

**Review by zhengruifeng — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @panbingkun and all.

**Comment by dongjoon-hyun** [comment]
Merged to master for Apache Spark 4.0.0.

**Pull request closed without merging**